docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -226,120 +231,233 @@ An example of these causes from the :ref:`large-scale-food fortification concept
+  terms of the all-cause mortality rate and the CSMRs of the green, blue, and
+  salmon causes:
 
-Hence, the mortality hazard for an individual in LBWSG category i is:  
+  .. math::
 
-mr_i
+    \sum_{\text{pink}}\text{CSMR} = \text{ACMR}
+    - \sum_{\text{salmon}}\text{CSMR}
+    - \sum_{\text{green}}\text{CSMR}
+    - \sum\limits_{\text{blue}}\text{CSMR}
 
-  | = ACMR_i
-  | = unaffected(unmodelled + modelled) + affected(unmodelled + modelled) x (1-PAF*) x :math:`rr_i`
+  This equation can be substituted into :eq:`mortality_hazard` below to
+  compute the mortality hazard for an individual simulant.
 
-= :math:`\sum\limits_{\text{pink}}\text{CSMR} + \sum\limits_{\text{salmon}}\text{EMR_state} + (\sum\limits_{\text{blue}}\text{CSMR}+\sum\limits_{\text{green}}\text{EMR_state})\cdot\text{(1-PAF*)}\cdot rr_i`
+To compute the individual mortality hazard :math:`\text{mr}(i)`, for the
+modelled causes (green and salmon) we will need the cause-state-dependent excess
+morality rates (EMR) instead of the CSMR, which is the average EMR over all
+cause states. For example, the excess mortality rates for a two-state cause
+(with condition / without condition) would be:
 
-.. important :: 
+   - mortality rate due to cause if the person does NOT have the condition: EMR=0
+   - mortality rate due to cause if the person HAS the condition: EMR of the condition
 
-   PAF in the above equation represents PAFs for most-detailed-cause (they are all roughly equal tp 0.94) affected by LBWSG (or as calculated in vivarium from LBWSG relative risks and exposure). This approach assumes that relative risks for LBWSG applies only to causes that GBD considers to be affected by LBWSG (green and blue causes). 
+We will need the following variables:
+
+.. math::
+  :nowrap:
+
+  \begin{align*}
+  &i &&= \text{identifier for an individual simulant}\\
+  &c &&= \text{identifier for a cause}\\
+  &\text{cat}(i) &&= \text{low birth weight short gestation category of individual $i$}\\
+  &\text{state}_c(i) &&= \text{current cause state of individual $i$ in cause model diagram for $c$}\\
+  &\text{CSMR}_c &&= \text{cause-specific mortality rate for cause $c$}\\
+  &\text{EMR}_{\text{state}_c(i)} &&= \text{excess mortality rate for the cause state state$_c(i)$}\\
+  &\textit{RR}_{\text{cat}(i)} &&= \text{relative risk for all-cause mortality in LBWSG category cat$(i)$}\\
+  &\text{PAF} &&= \text{PAF of LBWSG for affected causes at most-detailed cause level}
+  \end{align*}
+
+See the :ref:`note below <PAF information>` about how to compute the above PAF.
+Note that since :math:`\text{state}_c(i)` implicitly depends on the time
+:math:`t`, the individual mortality hazard will also depend on time. Using the
+above definitions, we will define the following individual mortality rates
+below:
+
+.. math::
+  :nowrap:
+
+  \begin{align*}
+  &\text{csmr}_c(i) &&= \text{unstratified cause-specific mortality hazard from cause $c$ for $i$}\\
```

Comment:
"unstratified" was initially confusing to me at first because I wasn't clear on what strata were being referenced (we always talk about age/sex strata, which are actually relevant here). Maybe "overall CSMR for a given age/sex/year/location group" would work?

Reply:
or we can say un-binned/uncategorized? (lbwsg bins/categories)

Reply:
Good points, I was struggling with what to call these. The name "overall CSMR for a given age/sex/year/location group" would not be accurate -- that's the capital CSMR's, whereas the lowercase csmr's are additionally stratified on what cause states the individual is in. Then the starred version is further stratified by what LBWSG category the individual is in, so I was trying to highlight that distinction, but I agree that "unstratified" is confusing.

Reply:
@alibow @Ninicorn How about "conditional cause-specific mortality hazard"? (see new commit)